### PR TITLE
changes so they work with assessment.peardeck.com

### PR DESCRIPTION
**Pull request by DoxrGitHub**
sorry I changed ading2210 to doxrgithub in some places but I'm too lazy to edit them back so pls change them back to ading2210

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ This script can bypass Edulastic's check for whether you're on the assignment or
 
 To use it, add this bookmarklet, or run fullscreen.js in the console: 
 ```js
-javascript: var r = new XMLHttpRequest(); r.open("GET", "https://cdn.jsdelivr.net/gh/ading2210/edulastic-tools@latest/fullscreen.js", true); r.addEventListener("load", function(){eval(this.responseText);}); r.send();
+javascript: var r = new XMLHttpRequest(); r.open("GET", "https://cdn.jsdelivr.net/gh/DoxrGitHub/edulastic-tools@latest/fullscreen.js", true); r.addEventListener("load", function(){eval(this.responseText);}); r.send();
```
Comment (ading2210) [inline README.md]: Can you revert all your changes on README.md?

File: fullscreen.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-var base_url = "https://raw.githubusercontent.com/ading2210/edulastic-tools/main"
+var base_url = "https://raw.githubusercontent.com/DoxrGitHub/edulastic-tools/main"
```
Comment (ading2210) [inline fullscreen.js]: Change this URL back please.

**Review by ading2210 — CHANGES_REQUESTED** [review]

File: fullscreen.js (hunk 2 above)
Comment (DoxrGitHub) [reply fullscreen.js]: done

File: README.md (hunk 1 above)
Comment (DoxrGitHub) [reply README.md]: okay

**Comment by Vincentvhe** [comment]
why fullscreen.js not worked

**Pull request merged**